scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -352,21 +353,59 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
         * Compute an id for the block with its content and its position.
         * Used as `track by` value, it allows the blocks to be well rendered.
         */
```

Comment:
It would be cool to update the documentation with what `index` is. I have mentioned this in my previous review too. This bit of logic seems a bit complex, so any documentation is very welcomed.